stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/ChatDomainImpl.kt

```diff
@@ -480,8 +439,8 @@ internal class ChatDomainImpl internal constructor(
         val cid = "%s:%s".format(channelType, channelId)
         if (!activeChannelMapImpl.containsKey(cid)) {
             val channelController = ChannelController(
-                mutableState = offlinePlugin.state.channel(channelType, channelId).toMutableState(),
```

Comment:
I didn't see the need for a container for `state` and `logic`. It would just add another class to SDK without much value, keeping the properties inside `ChatDomainImpl` seemed to be the best approach.